src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -94,7 +94,9 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
     currentVenueId: venueId,
   });
 
-  const [filterRelatedEvents, setFilterRelatedEvents] = useState(false);
+  const [filterRelatedEvents, setFilterRelatedEvents] = useState(
+    currentVenue?.id !== sovereignVenue?.id
+  );
```

Comment:
What happens if both are `undefined` ?

Reply:
will be `false` and it will work as normal.

Reply:
K, good enough. 

I've been making the venue/venueId optional for the purpose of 404 inside admin, just so there is NavBar visible